brax/envs/__init__.py

```diff
@@ -30,33 +30,79 @@
 from brax.envs import ur5e
 from brax.envs import wrappers
 
-_envs = {
-    'fetch': fetch.Fetch,
-    'ant': ant.Ant,
-    'grasp': grasp.Grasp,
-    'halfcheetah': halfcheetah.Halfcheetah,
-    'humanoid': humanoid.Humanoid,
-    'ur5e': ur5e.Ur5e,
-    'reacher': reacher.Reacher,
-    'reacherangle': reacherangle.ReacherAngle,
+_envs: Dict[str, Type[env.Env]] = {
+    "fetch": fetch.Fetch,
+    "ant": ant.Ant,
+    "grasp": grasp.Grasp,
+    "halfcheetah": halfcheetah.Halfcheetah,
+    "humanoid": humanoid.Humanoid,
+    "ur5e": ur5e.Ur5e,
+    "reacher": reacher.Reacher,
+    "reacherangle": reacherangle.ReacherAngle,
 }
 State = env.State
 Env = env.Env
 
 
 def create(env_name: str, **kwargs) -> Env:
-  """Creates an Env with a specified brax system."""
-  return _envs[env_name](**kwargs)
+    """Creates an Env with a specified brax system."""
```

Comment:
Nit: It looks like you're using some kind of linter with a different code style than ours - changes like this will be reverted by our internal tooling.  So for a cleaner git history, please avoid style changes re: tab widths, replacing ' with ", and so on.

Reply:
I see. I use black. Which auto-formatting tool do you recommend?

Reply:
I actually recommend avoiding auto-formatting!  This is formatted by internal Google tools that isn't public.  I'm sure you could probably customize an auto-formatter to better match this format, if that's helpful to you.

But the general principle I'd ask you to adhere to for now is to help us keep the blame history clean by avoiding changes that are unrelated to the PR, such as whole-file style changes that come from using an auto-formatter.

In the future as the project grows we could find some acceptable code style from something like yapf: https://github.com/google/yapf - I'll add an issue for it.

Reply:
> I actually recommend avoiding auto-formatting! 

I couldn't disagree more! :stuck_out_tongue: But that's a conversation for another time.

Good idea for the issue. I recommend `black`, or at least something completely unambiguous.

Reply:
Alright I've reverted the formatting changes to `brax/envs/__init__.py`.

Reply:
Thanks.  Definitely want to standardize on an easy to use formatter.  This is actually being actively discussed in the jax team, and we'll probably just follow their lead.

Reply:
So we don't forget: https://github.com/google/brax/issues/24